eng/common/pipelines/templates/steps/credscan.yml

```diff
@@ -8,6 +8,10 @@ steps:
 - pwsh: |
     if ("$(Build.Reason)" -eq 'PullRequest') {
       $targetBranch = "origin/$(System.PullRequest.TargetBranch)" -replace "refs/heads/"
+      git config core.quotepath off
```

Comment:
Lets add a comment linking to https://github.com/msysgit/msysgit/wiki/Git-for-Windows-Unicode-Support#disable-quoted-file-names so we know why we need these configurations.